app/models/user.rb

```diff
@@ -40,6 +42,8 @@ class User < ApplicationRecord
   scope :search_user, -> role, data {where("role = ? AND (phone_number = ? OR
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Metrics/AbcSize
**Severity**: INFO
**File**: app/models/user.rb L42

Assignment Branch Condition size for send_pin is too high. [15.43/15]

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*